cuegui/cuegui/Utils.py

```diff
@@ -416,8 +442,10 @@ def popupTail(file, facility=None):
 def popupView(file, facility=None):
     if not popupWeb(file, facility):
         from Constants import DEFAULT_EDITOR
-        JOB_LOG_CMD = str(QtGui.qApp.settings.value("LogEditor", DEFAULT_EDITOR))
-        shellOut("%s %s" % (JOB_LOG_CMD or DEFAULT_EDITOR, file))
+        editor = os.getenv('EDITOR', DEFAULT_EDITOR).split()
+        job_log_cmd = QtGui.qApp.settings.value("LogEditor", editor) or editor
```

Comment:
nit: I know it's not new but I'm not a huge fan of the `or` here - this looks like a three-choice series that would be more clear in an if-elif-else block.

Reply:
Yeah, this section felt a bit messy and looking at it again.
`job_log_cmd = QtGui.qApp.settings.value("LogEditor", editor) or editor`
The second parameter of the `value` method is the default value, so this `or` isn't doing anything anyways. I'm cleaning it up now.